plugins/color_panel/macos/FLEColorPanelPlugin.mm

```diff
@@ -33,7 +33,12 @@ - (NSString *)channel {
 - (void)handleMethodCall:(FLEMethodCall *)call result:(FLEMethodResult)result {
   BOOL handled = YES;
   if ([call.methodName isEqualToString:@(plugins_color_panel::kShowColorPanelMethod)]) {
-    [self showColorPanel];
+    BOOL showAlpha = YES;
+    if ([call.arguments isKindOfClass:[NSDictionary class]]) {
+      showAlpha = [[call.arguments valueForKey:@(plugins_color_panel::kColorPanelShowAlpha)]
+                   boolValue];
```

Comment:
The wrapping here is a bit odd; is this clang-format output, or manual/Xcode?

(If it's clang-format that's fine, I just thought it would do something a bit different, and would like everything to be clang-formatted since it avoids churn on later CLs where people format.)

Reply:
That was me. Fixed with clang-format.